develop/items/custom-enchantment-effects.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Custom Enchantment Effects
+description: Learn how to create your enchantment effects.
+authors:
+  - krizh-p
+---
+
+# Custom Enchantments {#custom-enchantments}
+
+Starting from version 1.21, custom enchantments in Minecraft use a "data-driven" approach. This makes it easier to add simple enchantments, like increasing attack damage, but more challenging to create complex ones. The process involves breaking down enchantments into _effect components_.
```

Comment:
> Starting from version 1.21

Does this have to be mentioned here? The plugin already highlights it.

> in Minecraft

Again, the entire Fabric project revolves around Minecraft, no need to specify it in every page IMO.